### PR TITLE
Fix: added html_entity_decode for images

### PR DESCRIPTION
**Pull request by Grumvol**

### CODE REVIEW

File: upload/extension/opencart/catalog/controller/module/carousel.php [hunk 1]
```diff
@@ -16,8 +16,8 @@ public function index($setting) {
 		$results = $this->model_design_banner->getBanner($setting['banner_id']);
 
 		foreach ($results as $result) {
-			if (is_file(DIR_IMAGE . $result['image'])) {
-				$data['banners'][] = [
+			if (is_file(DIR_IMAGE . html_entity_decode($result['image'], ENT_QUOTES, 'UTF-8'))) {
+				$data['banners'][] = array(
```
Comment (eka7a) [inline upload/extension/opencart/catalog/controller/module/carousel.php:20]: Change array to [] 53553cc57ec0e701dbbfcf77f04792b58bca76ca

Comment (Grumvol) [reply upload/extension/opencart/catalog/controller/module/carousel.php]: done

**Comment by Grumvol** [comment]
#8269

File: upload/extension/opencart/catalog/controller/module/carousel.php (hunk 1 above)
Comment (Grumvol) [reply upload/extension/opencart/catalog/controller/module/carousel.php]: #8269

**Pull request closed without merging**